examples/OpticalTracker/README.md

```diff
@@ -0,0 +1,16 @@
+Proximity Focusing RICH
+=======================
+
+Example for `Geant4OpticalTracker` SD plugin usage
+
+Usage:
+```bash
+scripts/build.sh      # cmake build+install
+scripts/view.sh       # produce TGeo file, view in JSroot
+scripts/run_npsim.sh  # throw particles with npsim
+```
```

Comment:
Can we get instructions on how to display a photon ring?

Reply:
added to `README.md`